sysl2/sysl/cmd_runner.go

```diff
@@ -37,6 +51,8 @@ func (r *cmdRunner) Run(which string, fs afero.Fs, logger *logrus.Logger) error
 }
 
 func (r *cmdRunner) Configure(app *kingpin.Application) error {
+	app.UsageTemplate(kingpin.SeparateOptionalFlagsUsageTemplate)
```

Comment:
this line should be when the app is created

Reply:
moved to `sysl.go` in `main3()`